.buildkite/rootfs_images/rootfs_utils.jl

```diff
@@ -74,12 +88,59 @@ function debootstrap(name::String; release::String="buster", variant::String="mi
         end
         run(`sudo chroot --userspec=$(getuid()):$(getgid()) $(rootfs) locale-gen`)
     end
+end
 
-    # Archive it into a `.tar.gz` file
-    @info("Archiving", tarball_path, artifact_hash)
-    archive_artifact(artifact_hash, tarball_path)
+# Helper structure for installing alpine packages that may or may not be part of an older Alpine release
+struct AlpinePackage
+    name::String
+    repo::Union{Nothing,String}
 
-    return tarball_path
+    AlpinePackage(name, repo=nothing) = new(name, repo)
+end
+function repository_arg(repo)
+    if repo === nothing
+        return String[]
+    end
+    if startswith(repo, "https://")
+        return ["--repository=$(repo)"]
+    end
+    return ["--repository=http://dl-cdn.alpinelinux.org/alpine/$(repo)/main"]
+end
```

Comment:
```suggestion
function repository_arg(repo)
    if startswith(repo, "https://")
        return ["--repository=$(repo)"]
    end
    return ["--repository=http://dl-cdn.alpinelinux.org/alpine/$(repo)/main"]
end
repository_arg(::Nothing) = String[]
```

Reply:
I think nothing checks like this tend to be inline in the main function.